src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -87,6 +87,8 @@ typedef struct MPIDI_OFI_am_header_t {
     uint64_t payload_sz:MPIDI_OFI_AM_PAYLOAD_SZ_BITS;   /* data size on this OFI message. This
                                                          * could be the size of a pipeline segment
                                                          * */
+    uint8_t vni_src;
+    uint8_t vni_dst;
```

Comment:
Is there a way to tell the destination vni without having to include it in the AM header? Like, isn't that implicitly known when reading the completion queue?

Reply:
The `vni_dst` is known since we are poking the particular cq context. The `vni_src` is not known and it is important to reply to an am message. While we can omit `vni_dst`, I find it easier to maintain by including both.

Reply:
Got it. I think a comment explaining the rationale (ease of maintenance) would help readers of the code. If there are size concerns with this header, then we can properly weigh the tradeoff at some later time.

Reply:
Comment added.